### ISSUE TITLE
Every fire extinguisher I find is empty

### ISSUE THREAD

**Issue by Duck13Tales** [CleverRaven/Cataclysm-DDA#41848]
I have found more than 10 fire extinguishers but all of them have 0 extinguishing agent. Is it intended?

# Steps To Reproduce

Just wander through cities scavenging till you find fire extinguishers around. Be it in basements, stores, offices, houses etc, they are always completely empty.

# Expected behavior

I am not sure if it is intended.

# Screenshots

Not applicable.

# Versions and configuration

I am using using the CDDA 0.E-b10798 with the mods:

DDA
Disable NPC needs
Stats through skills
Stats through kills
SpeedyDex

**Comment by LucasHoage** [comment]
I was able to fix this in a very unproductive way that would require updating every single mention by hand. Not a solution.

items: [
{ "item": "extinguisher", "prob": 100, "charges-min": 0, "charges-max": 100 },

Works inside of an itemgroup. I tried recreating the cell phone fix from earlier, but that did not work and threw a fit about fitting a plastic bottle inside a fire extinguisher (extinguisher agent uses a plastic bottle container)
